app/assets/javascripts/admin/addon/templates/config-flags-settings.hbs

```diff
@@ -0,0 +1,5 @@
+<AdminConfigAreas::FlagsSettings
```

Comment:
I didn't make a component like this for the backups one, maybe I should. I just used the old controller pattern

Reply:
Do you think I should change?